projects/plugins/wpcomsh/notices/plan-notices.php

```diff
@@ -86,22 +86,22 @@ function ( $purchase1, $purchase2 ) {
 		$plan_messages = array(
 			/* translators: %1$s is a link for plan renewal, %2$s human readable time e.g. January 1, 2021, %3$s site URL */
 			'personal'  => __(
-				'The Starter plan for <strong>%3$s</strong> expires on %2$s. <a href="%1$s">Renew your plan</a> to retain Starter plan features such as 6 GB storage space, no WordPress.com ads, and Subscriber-only content.',
+				'The Personal plan for <strong>%3$s</strong> expires on %2$s. <a href="%1$s">Renew your plan</a> to retain Personal plan features such as 6 GB storage space, no WordPress.com ads, and Subscriber-only content.',
 				'wpcomsh'
 			),
 			/* translators: %1$s is a link for plan renewal, %2$s human readable time e.g. January 1, 2021, %3$s site URL */
 			'premium'   => __(
-				'The Explorer plan for <strong>%3$s</strong> expires on %2$s. <a href="%1$s">Renew your plan</a> to retain Explorer plan features such as site monetization, VideoPress, and Google Analytics support.',
+				'The Premium plan for <strong>%3$s</strong> expires on %2$s. <a href="%1$s">Renew your plan</a> to retain Premium plan features such as site monetization, VideoPress, and Google Analytics support.',
 				'wpcomsh'
 			),
 			/* translators: %1$s is a link for plan renewal, %2$s human readable time e.g. January 1, 2021, %3$s site URL */
 			'business'  => __(
-				'The Creator plan for <strong>%3$s</strong> expires on %2$s. <a href="%1$s">Renew your plan</a> to retain Creator plan features such as custom plugins and themes, SFTP, and phpMyAdmin access.',
+				'The Business plan for <strong>%3$s</strong> expires on %2$s. <a href="%1$s">Renew your plan</a> to retain Business plan features such as custom plugins and themes, SFTP, and phpMyAdmin access.',
 				'wpcomsh'
 			),
 			/* translators: %1$s is a link for plan renewal, %2$s human readable time e.g. January 1, 2021, %3$s site URL */
 			'ecommerce' => __(
-				'The Entrepreneur plan for <strong>%3$s</strong> expires on %2$s. <a href="%1$s">Renew your plan</a> to retain Entrepreneur plan features such as custom plugins and themes, SFTP, and phpMyAdmin access.',
+				'The Commerce plan for <strong>%3$s</strong> expires on %2$s. <a href="%1$s">Renew your plan</a> to retain Commerce plan features such as custom plugins and themes, SFTP, and phpMyAdmin access.',
```

Comment:
I'm just double-checking. We [used to call this the "eCommerce" plan](https://github.com/Automattic/wpcomsh/pull/1640/files#diff-3e4a9da37d078c4f7e1aedac1b5ed806f48e886d1bee6a2d6759ceabb10c4fffL104). Is "Commerce" correct?

I suspect we need to change the names [here](https://github.com/Automattic/jetpack/blob/cac2b32d36df1d9fb4aa674ba9a9a3fdef63d746/projects/plugins/wpcomsh/notices/plan-notices.php#L119-L137) as well?

Reply:
> I suspect we need to change the names [here](https://github.com/Automattic/jetpack/blob/cac2b32d36df1d9fb4aa674ba9a9a3fdef63d746/projects/plugins/wpcomsh/notices/plan-notices.php#L119-L137) as well?

Thanks for catching that. I updated there as well.

> We [used to call this the "eCommerce" plan](https://github.com/Automattic/wpcomsh/pull/1640/files#diff-3e4a9da37d078c4f7e1aedac1b5ed806f48e886d1bee6a2d6759ceabb10c4fffL104). Is "Commerce" correct?

Confirming that the name is "Commerce". We made this change in early 2023 and looks like that change was not implemented here. Thanks for bringing it up!